account_move_report/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 {
     "name": "Journal Entries Report",
-    "version": "saas~17.0.0.1",
+    "version": "11.0.0.0.1",
```

Comment:
I think that bust be `11.0.1.0.0`

Is correct @moylop260 ?

Reply:
yep

Reply:
More info about:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

Reply:
@keylor2906 

Could you make the improvements, please?

Regards